clang/test/CheckedCRewriter/complex_expressionBUG.c

```diff
@@ -0,0 +1,58 @@
+// Tests for Checked C rewriter tool.
```

Comment:
Why is this file failing, can you explain? On other conversation threads I understood that failures related to missing `malloc` type parameters, but there is no `malloc` here.

Reply:
Sorry, this is my bad. This isn't due to malloc, but rather due to #160 , just like `ptrint.c`, where a checked pointer being cast to an unchecked one causes the compiler to yell at us.

Reply:
So, is this a new issue, or is it #160

Reply:
Sorry, just realized a thumbs up doesn't answer the question. This is #160